tests/mlio-test/test_image_reader.cxx

```diff
@@ -0,0 +1,445 @@
+#include <gtest/gtest.h>
+#include <mlio.h>
+
+#ifdef MLIO_BUILD_IMAGE_READER
```

Comment:
This file is only used if `MLIO_BUILD_IMAGE_READER` is defined. Instead of using `#ifdef`, I recommend managing this in CMake. Here is an example:
https://github.com/awslabs/ml-io/blob/master/src/mlio/CMakeLists.txt#L83

Reply:
the quoted line 83 has `shuffled_instance_reader.cxx` - is that what you meant to refer to?